docs/users/cli-usage.md

```diff
@@ -29,6 +29,109 @@ Available Commands:
+
+After you've run `ddev config`, you're ready to start running your site. To start running ddev, simply enter:
+```
+ddev start
+``` 
+When running `ddev start` you should see output informing you that the site's environment is being started. If startup is successful, you'll see a message like the one below telling you where the site can be reached.
+```
+Successfully started wordpress_site
+Your application can be reached at: http://wordpress_site.ddev.local
+```
+Quickstart instructions regarding database imports, can be found under [Database Imports](#database-imports).
+### Drupal 7 Quickstart
+Beginning to use ddev with a Drupal 7 site is as simple as cloning the site's repository and checking out its directory.
+```
+git clone https://github.com/user/my_drupal7_site
+cd my_drupal7_site
+```
+Now to start working with ddev. Inside of your site's working directory, enter the following command:
+```
+ddev config
+```
+
+_Note: ddev config will prompt you for a site name, docroot, and app type._
+
+After you've run `ddev config` you're ready to start running your site. Run ddev using a simple:
+```
+ddev start
+``` 
+When running `ddev start` you should see output informing you that the site's environment is being started. If startup is successful, you'll see a message like the one below telling you where the site can be reached.
+```
+Successfully started my_drupal7_site
+Your application can be reached at: http://my_drupal7_site.ddev.local
+```
+Quickstart instructions regarding database imports, can be found under [Database Imports](#database-imports).
+### Drupal 8 Quickstart
+You can get started with Drupal 8 sites on ddev either by cloning a git repository or using a new or existing composer project.
+
+**Git Clone Example**
+```
+git clone https://github.com/user/my_drupal8_site
+cd my_drupal8_site
+```
+
+**Composer Setup Example**
+```
+composer create-project drupal-composer/drupal-project:8.x-dev my_drupal8_site --stability dev --no-interaction
+cd my_drupal8_site
+```
+_You can find more information on composer and how to use it [here](https://github.com/drupal-composer/drupal-project)._
+
+The next step is to configure ddev. Inside of your site's working directory, enter the following command:
+```
+ddev config
+```
+
+_Note: ddev config will prompt you for a site name, docroot, and app type._
+
+After you've run `ddev config` you're ready to start running your site. Run ddev using a simple:
+```
+ddev start
+``` 
+When running `ddev start` you should see output informing you that the site's environment is being started. If startup is successful, you'll see a message like the one below telling you where the site can be reached.
+```
+Successfully started my_drupal8_site
+Your application can be reached at: http://my_drupal8_site.ddev.local
+```
+**Next:** Learn how to import a database in the [Database Imports](#database-imports) section.
+#### Database Imports
```

Comment:
Probably worthy of keeping as the same indent level as the WordPress/Drupal guides. ###

Reply:
This section could potentially be linked out to https://ddev.readthedocs.io/en/latest/users/cli-usage/#importing-a-database, which is fairly robust. This would be similar to how we link out the install prereqs.